src/auth/jwt-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface JwtPayload {
+  userId: string;
+  email: string;
```

Comment:
PullProofAI

A new field 'role' has been added to the JwtPayload interface. This change enhances the JWT payload by including user roles, which can be useful for implementing role-based access control or authorization logic.
```diff
+  role: string;
```


<!-- This is an auto-generated comment by PullProofAI -->